documentation/experimental_features.md

```diff
@@ -8,6 +8,30 @@ API might change, requiring the user to update their code or configuration. The
 Bolt team attempts to make these changes painless by providing useful warnings
 around breaking behavior where possible. 
 
+## Configuration policies and `bolt policy` subcommand
+
+This feature was introduced in [Bolt 3.21.0](). 
+
+Configuration policies are public classes that can be assigned directly to one
```

Comment:
I think it's important to include in this top paragraph that policies are only different from classes in that they are listed in `bolt-project.yaml` under the `policies` key. Something like:
```
Configuration policies are public classes that can be assigned directly to one or more nodes via the Bolt command line. A class becomes a policy when it's listed under the `policies` key in `bolt-project.yaml`, and if the `policies` key is empty then no policies will be available for Bolt to apply.
```